libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -59,10 +59,16 @@ var ec2MetadataFetcher = provider{
 
 	Create: func(_ string, config *conf.C) (metadataFetcher, error) {
 		ec2Schema := func(m map[string]interface{}) mapstr.M {
-			m["service"] = mapstr.M{
-				"name": "EC2",
+			cloud := mapstr.M{
```

Comment:
I suggest to rename the map name to something like `meta` because if it is named `cloud` , to me it is confusing that it only includes the cloud object.

Reply:
done https://github.com/elastic/beats/pull/37651/commits/2765695b9cb165263e7433342fcd5e52df0ba942